build-system/tasks/integration.js

```diff
@@ -19,14 +19,25 @@ const argv = require('minimist')(process.argv.slice(2));
 const fs = require('fs-extra');
 const globby = require('globby');
 const log = require('fancy-log');
+const pathModule = require('path');
 const {
   RuntimeTestRunner,
   RuntimeTestConfig,
 } = require('./runtime-test/runtime-test-base');
+const {buildNewServer} = require('../server/typescript-compile');
 const {buildRuntime} = require('../common/utils');
-const {cyan, green} = require('ansi-colors');
+const {cyan, red} = require('ansi-colors');
 const {maybePrintArgvMessages} = require('./runtime-test/helpers');
 
+const INTEGRATION_FIXTURES = [
+  './test/fixtures/**/*.html',
+  '!./test/fixtures/e2e',
```

Comment:
FMI: What does `!.` mean here?

Reply:
! at the beginning of a pattern will negate the match.

See "Globbing Patterns" at https://github.com/sindresorhus/globby.

Good question though, had to verify.